ot-jars/README.md

```diff
@@ -0,0 +1,20 @@
+# Introduction
+If you read this you probably want to publish a new version of OT jars image.
+In this document you can find instructions how to publish the image.
+
+# Location
+The image is hosted in Docker Hub in sumologic organization https://hub.docker.com/r/sumologic/opentelemetry-jars 
+
+# Version history
+- `v0.0.1` - contains OT jars version `0.3.0`
+- `v0.0.2` - contains OT jars version `0.6.0`
```

Comment:
Just a thought - perhaps we could sync release numbers with OT release numbers? So `v0.6.0` or `v0.6.0.0` for the most recent jars?

Reply:
Good idea, I will do that for next releases of OT jars.